x/incentives/client/cli/query.go

```diff
@@ -384,31 +386,80 @@ $ %s query incentives rewards-estimation
 				return err
 			}
 
-			lockIdStrs := strings.Split(lockIdsCombined, ",")
-			lockIds := []uint64{}
-			for _, lockIdStr := range lockIdStrs {
-				lockId, err := strconv.ParseUint(lockIdStr, 10, 64)
+			var res *lockuptypes.AccountLockedLongerDurationResponse
```

Comment:
Consider  declaring and initializing on line 390 like:
```
res, err := ...
```

Reply:
Sorry, can you explain further? My intent was to only initialize if the owner flag is present. I'm sure the way you are explaining is better, I just don't quite get what you are suggesting

Reply:
If that was the intent, I think it should be fine. I misinterpreted it, sorry